### PR TITLE
Implement Error Band

### PR DESCRIPTION
**Pull request by chanwutk**

### CODE REVIEW

**Comment by chanwutk** [comment]
@kanitw At line 135 of errorbar.ts, I put `any` as a return type because it gives me an error if otherwise. What should be the correct type to put in as the return type?

**Comment by kanitw** [comment]
`transform` is `Transform[]`.  The rest of them, you should be able to hover each of the variable and see the type.  

(ps. I look at it and I'm surprised TS isn't smart enough to figure this out too --weird!)

**Comment by chanwutk** [comment]
I forgot to do the #3770, I will do it in the next commit.

File: src/compositemark/errorband.ts [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+import {ErrorBarCenter, ErrorBarExtent, errorBarParams} from './errorbar';
+
+export const ERRORBAND: 'errorband' = 'errorband';
+export type ErrorBand = typeof ERRORBAND;
+
+export type ErrorBandPart = 'band' | 'rims';
+
+const ERRORBAND_PART_INDEX: Flag<ErrorBandPart> = {
+  band: 1,
+  rims: 1
```
Comment (kanitw) [inline src/compositemark/errorband.ts]: @domoritz any thoughts about the name `rims`? 

A few other alternatives: edges, borders

Comment (domoritz) [reply src/compositemark/errorband.ts]: outline, limits

I think border(s) is probably the most intuitive

**Comment by chanwutk** [comment]
Since we have `borders` for `error band` and `ticks` for `error bar`, should we find a word that can substitute both `borders` and `ticks` but still makes sense, so that both `error bar` and `error band` have the same syntax?

**Comment by domoritz** [comment]
I'm not too worried about the difference but I'm open to suggestions.

**Comment by chanwutk** [comment]
How about edges or ends?

**Comment by domoritz** [comment]
Edge doesn't work for error bar, ends doesn't work for error band, I think.

**Comment by kanitw** [comment]
I like `borders` too

**Pull request merged**